code/modules/clothing/outfits/ert.dm

```diff
@@ -157,7 +162,10 @@
 		/obj/item/melee/baton/loaded=1,\
 		/obj/item/clothing/mask/gas/sechailer/swat=1,\
 		/obj/item/gun/energy/pulse/pistol/loyalpin=1,\
```

Comment:
They literally already have a sidearm. Why do they need a second one?

Reply:
pluse is lethal only, mini-eguns have non-lethal settings

Reply:
As already discussed, no it isn't.